luis-delete.js

```diff
@@ -0,0 +1,40 @@
+function del(){
+    var rp = require('request-promise');
+    var fse = require('fs-extra');
+    var path = require('path');
+    const LUIS_programmaticKey = "67fa05fd36ca4d4cbc8c2eb91e41dc10";
+    const LUIS_appId = "6a18e3d5-7267-4b94-b1c4-8944b866fbc8";
+    const LUIS_versionId = "0.1";
+    this.deleteUtt =function(exampleId){
+        const LUIS_exampleId =exampleId;
+        var configAddUtterance = {
```

Comment:
`var` is old javascript variable notation, in new versions, javascript supports `let` and `const`. Better to use these two to declare new variables, because these two syntax has a better scope control. 

`let` is similar to var, `const` is static value, once you declare it, you can not change it.